data/ShreyashSomvanshi.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Shreyash Somvanshi",
+  "type": "personal",
+  "bio": "Artificial Intelligence and Data Science Engineer | Open Source Beginner | Technical Content Writer ",
+  "tags": ["Data Science", "AI", "Open Source", "Technical Blogging"],
+  "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/ShreyashSomvanshi" },
+    { "icon": "FaLinkedIn" , "url": "https://linkedin.com/in/shreyash-somvanshi-859893215/"},
```

Comment:
please change the icon name to `FaLinkedin`

Reply:
Ok I'll just change it. Thank You!